### PR TITLE
fix(#1761): explain require next <cr> and space ending problem

### LINKED ISSUE #1761: Solve problems in SQL commands
I find two problems when using the explain+sql commands via terminal

(1) Need to click the "enter" key twice before seeing the explain results:

127.0.0.1:6527/test> explain select * from test;
127.0.0.1:6527/test> 	DATA_PROVIDER(request=test)

(2) Need to split the sql with the semicolon (;) with a character (e.g., an enter key or space), otherwise it will report errors:

127.0.0.1:6527/test> select * from (select * from
                  ->  window win1 as (partition by c1 order by c2 rows_range between 172799999 preceding and 0s preceding MAXSIZE 1000),
                  -> win2 as (partition by c1 order by c2 rows_range between 172799999 preceding and 0s preceding MAXSIZE 1000))
                  -> as out0
                  ->  last join (select * from test t1 last join test t2 on t1.c2=t2.c2) as out1 on out0.c2=out1.c2; 
                  -> 
                  -> ;
Error: Syntax error: Expected end of input but got ";" [at 6:1]

127.0.0.1:6527/test> select * from (select * from
                  ->  window win1 as (partition by c1 order by c2 rows_range between 172799999 preceding and 0s preceding MAXSIZE 1000),
                  -> win2 as (partition by c1 order by c2 rows_range between 172799999 preceding and 0s preceding MAXSIZE 1000))
                  -> as out0
                  ->  last join (select * from test t1 last join test t2 on t1.c2=t2.c2) as out1 on out0.c2=out1.c2
                  -> ;
 ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- 
  c1   c2   c3   c6   c7   c1   c2   c3   c6   c7   c1   c2   c3   c6   c7  
 ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----

### PR DESCRIPTION
**Pull request by aceforeverd**
close #1761 

two problem in cli:
1. explain result missing a `<CR>`
2. fail to precess ending space

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: src/cmd/sql_cmd.h [hunk 1]
```diff
@@ -73,7 +94,7 @@ void HandleSQL(const std::string& sql) {
                 while (result_set->Next()) {
                     std::string val;
                     result_set->GetAsString(0, val);
-                    std::cout << val;
+                    std::cout << val << std::endl;
```
Comment (aceforeverd) [inline src/cmd/sql_cmd.h]: check `desc table`

File: src/cmd/sql_cmd_test.cc [hunk 2]
```diff
@@ -1969,6 +1969,27 @@ TEST_P(DBSDKTest, DeployStatsOnlyCollectDeployProcedure) {
     }
 }
 
+class StripSpaceTest : public ::testing::TestWithParam<std::pair<std::string_view, std::string_view>> {};
+
+std::vector<std::pair<std::string_view, std::string_view>> strip_cases = {
```
Comment (vagetablechicken) [inline src/cmd/sql_cmd_test.cc:1974]: how about "show\ncomponents\n;"

Comment (aceforeverd) [reply src/cmd/sql_cmd_test.cc:1974]: statement str inside is handle by zetasql, should be fine.

anyway case like `show components; set val = ';    '` probably not work as expect

**Comment by aceforeverd** [comment]
[Bugs (Normal)](https://trello.com/c/WhbjlRN2/69-bugs-normal)

**Review by dl239 — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**